src/assets/i18n/en.json

```diff
@@ -258,5 +258,7 @@
     "Sub_event2": "Class representatives are taking the initiative to regulary organize some ecents within the major (cinema, laser game, etc.) in order to give students the opportunity to connect with each others beyond the professional environment.",
     "Sub_event3": "To establish a culture of team cohesion and in order to boost up the dialogue between teams, moments of exchange and relaxation bringing together teachers and students are organised",
     "Title_event4": "SIGL's Disintegration",
-    "Sub_event4": "In order to celebrate together the end of studies and the success of their year, a disintegration event is organized by the students at the end of their courses. Lastest destination is: Lisbon!"
+    "Sub_event4": "In order to celebrate together the end of studies and the success of their year, a disintegration event is organized by the students at the end of their courses. Lastest destination is: Lisbon!",
+    "Title_event5": "SIGL's 2024 End of year picture",
+    "Sub_event5": "The end of the year is always a good opportunity to take a picture of the whole SIGL."
```

Comment:
Ok  I do not see any description on the Trello ticket so I imagine that you imagine it ? Can you please confirm it with the product owner please ?